openbas-api/src/test/java/io/openbas/rest/ExerciseLessonsApiTest.java

```diff
@@ -0,0 +1,133 @@
+package io.openbas.rest;
+
+import io.openbas.IntegrationTest;
+import io.openbas.database.model.*;
+import io.openbas.database.repository.ExerciseRepository;
+import io.openbas.database.repository.LessonsCategoryRepository;
+import io.openbas.database.repository.TeamRepository;
+import io.openbas.database.repository.UserRepository;
+import io.openbas.rest.exercise.ExerciseService;
+import io.openbas.rest.lessons.form.LessonsSendInput;
+import io.openbas.service.MailingService;
+import io.openbas.utils.mockUser.WithMockPlannerUser;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.time.Instant;
+import java.util.List;
+
+import static io.openbas.database.model.ExerciseStatus.SCHEDULED;
+import static io.openbas.rest.exercise.ExerciseApi.EXERCISE_URI;
+import static io.openbas.utils.JsonUtils.asJsonString;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.mockito.Mockito.*;
+
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ExerciseLessonsApiTest extends IntegrationTest {
+
+  static Exercise EXERCISE;
+  static LessonsCategory LESSONCATEGORY;
+  static Team TEAM;
+  static User USER;
+
+  @Autowired
+  private MockMvc mvc;
+  @Autowired
+  private ExerciseService exerciseService;
+  @Autowired
+  private ExerciseRepository exerciseRepository;
+  @Autowired
+  private LessonsCategoryRepository lessonsCategoryRepository;
+  @SpyBean
+  private MailingService mailingService;
+  @Autowired
+  private TeamRepository teamRepository;
+  @Autowired
+  private UserRepository userRepository;
+
+  @BeforeAll
+  void beforeAll() {
+    EXERCISE = getExercise();
+    LESSONCATEGORY = getLessonCategory();
+  }
+
+  @AfterAll
+  void afterAll() {
+    this.exerciseRepository.delete(EXERCISE);
+    this.lessonsCategoryRepository.delete(LESSONCATEGORY);
+    this.teamRepository.delete(TEAM);
+    this.userRepository.delete(USER);
+  }
+
+
+  private LessonsCategory getLessonCategory() {
```

Comment:
as is done with team or exercise this function should be inside the openbas-api/src/test/java/io/openbas/utils/fixtures folder